### ISSUE TITLE
KITTI dataset

### ISSUE THREAD

**Issue by liusiping10** [JunjH/Visualizing-CNNs-for-monocular-depth-estimation#4]
Could you provide the KITTI dataset in the same format as the NYU-v2? thanks a lot.

**Comment by savasozkan** [comment]
Could you shared the KITTI pretrained models? Best.

**Comment by JunjH** [comment]
Please check this link to download the related code on validation set of kitti. 
https://drive.google.com/file/d/1QLqhl5Imr8s3y7GVTdBjYbXX3AZ_hOoH/view?usp=sharing

You can download the whole dataset from the official website.
